[2025-04-22T00:25:06.400+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-22T00:25:06.405+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_stock_price.load_data manual__2025-04-22T00:25:05.902618+00:00 [queued]>
[2025-04-22T00:25:06.408+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_stock_price.load_data manual__2025-04-22T00:25:05.902618+00:00 [queued]>
[2025-04-22T00:25:06.408+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-04-22T00:25:06.412+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load_data> on 2025-04-22 00:25:05.902618+00:00
[2025-04-22T00:25:06.415+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'daily_stock_price', 'load_data', 'manual__2025-04-22T00:25:05.902618+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/daily_stock_price.py', '--cfg-path', '/tmp/tmpys6i3q6o']
[2025-04-22T00:25:06.417+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=349) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-22T00:25:06.417+0000] {standard_task_runner.py:105} INFO - Job 150: Subtask load_data
[2025-04-22T00:25:06.417+0000] {standard_task_runner.py:72} INFO - Started process 352 to run task
[2025-04-22T00:25:06.434+0000] {task_command.py:467} INFO - Running <TaskInstance: daily_stock_price.load_data manual__2025-04-22T00:25:05.902618+00:00 [running]> on host fbe94ad2eb9c
[2025-04-22T00:25:06.461+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_stock_price' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T00:25:05.902618+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-22T00:25:05.902618+00:00'
[2025-04-22T00:25:06.462+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-22T00:25:06.467+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-04-22T00:25:06.468+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-04-22T00:25:06.468+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-04-22T00:25:07.260+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:25:07.262+0000] {logging_mixin.py:190} INFO - YF.download() has changed argument auto_adjust default to True
[2025-04-22T00:25:08.080+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:25:09.555+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:25:09.557+0000] {logging_mixin.py:190} INFO - Successfully loaded /tmp/TSLA_data.csv into ANALYTICS.stock_prices
[2025-04-22T00:25:09.752+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:25:09.994+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:25:11.251+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-04-22T00:25:11.252+0000] {logging_mixin.py:190} INFO - Successfully loaded /tmp/SPY_data.csv into ANALYTICS.stock_prices
[2025-04-22T00:25:11.252+0000] {connection.py:788} INFO - closed
[2025-04-22T00:25:11.290+0000] {connection.py:794} INFO - No async queries seem to be running, deleting session
[2025-04-22T00:25:11.435+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-22T00:25:11.450+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-22T00:25:11.450+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=daily_stock_price, task_id=load_data, run_id=manual__2025-04-22T00:25:05.902618+00:00, execution_date=20250422T002505, start_date=20250422T002506, end_date=20250422T002511
[2025-04-22T00:25:11.465+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-22T00:25:11.477+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-22T00:25:11.477+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
